Example/Tests/Resources/Responses/start_with_variant_debug_info_response.json

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
autoformat this json file

Reply:
sadly, this is the automatic xcode format. I'll do it through another IDE for this